tests/test_data.py

```diff
@@ -0,0 +1,105 @@
+# Author: Kelly Wu
+# Date: 2023-01-19
+from mercedestrenz.data import listing_search
+import pandas as pd
+import numpy as np
+
+def test_listing_search():
+    """Unit tests for listing_search function. """
+    
+    # Create a fictitious dataset for testing
+    data = {'url':['www.a.ca', 'auto.com', 'benz.ca', 'cars.ca'],
+        'condition':["good", 'average', 'average', 'new'],
+        'price_USD' : [50000, 20000, 80000, 90000],
+        'model' : ['s-class', 'glk', 'gls', 'gls'],
+        'odometer': [12000, 30000, 35000, 150000],
+        'year': [2016, 2015, 2020, 2010]
+        }
```

Comment:
- remove `url` column
- use updated model types
- update `odometer` to `odometer_mi`
```suggestion
    data = {'url':['www.a.ca', 'auto.com', 'benz.ca', 'cars.ca'],
        'condition':["good", 'average', 'average', 'new'],
        'price_USD' : [50000, 20000, 80000, 90000],
        'model' : ['s-class', 'gl-class', 'm-class', 's-class'],
        'odometer_mi': [12000, 30000, 35000, 150000],
        'year': [2016, 2015, 2020, 2010]
        }
```